auth-foundation/src/main/java/com/okta/authfoundation/credential/SecurityOptions.kt

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2024-Present Okta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.okta.authfoundation.credential
+
+import com.okta.authfoundation.AuthFoundationDefaults
+import kotlinx.serialization.Serializable
+
+@Serializable
+sealed interface SecurityOptions {
+    val keyAlias: String
+    val encryptionAlgorithm: String
+    val userAuthenticationRequired: Boolean
+
+    data class Default(
```

Comment:
I see that there are a set of discrete options. Do developers have the ability to define their own values for this? For example, in Swift the [Credential.Security.standard](https://okta.github.io/okta-mobile-swift/development/authfoundation/documentation/authfoundation/credential/security/standard) value is a `static var`, meaning a developer can reassign the default value at runtime. 

Consider the following:

```swift
class MyApp {

  func applicationDidFinishLaunching() {
    Credential.Security.standard = [ .accessGroup("com.myapp.commonAccessGroup"), .accessibility(.afterFirstUnlock) ]
  }
}
```

This will allow the developer later to say

```swift
let credential = try Credential.store(token)
```

instead of having to use the following everywhere

```swift
let credential = try Credential.store(token, security: [ .accessGroup("com.myapp.commonAccessGroup"), .accessibility(.afterFirstUnlock) ])
```

Reply:
Good point. Yes, I will add this.